test/mailers/previews/notifier_preview.rb

```diff
@@ -1,7 +1,12 @@
 class NotifierPreview < ActionMailer::Preview
   def digest
-    recipients = User.all.pluck :email
-    options = {recipients: recipients, completed: 0, remaining: 3}
+    options = {
+      recipients: ['vasyllasiak@interlink-ua.com'],
+      completed_todos: {List.last => [Item.first, Item.last]},
```

Comment:
@vlasiak, I hope both items belong to the same list (to the last list), otherwise it might be confusing. Anyway I suppose that given class will be deleted within #19, right?